### ISSUE TITLE
It adds new refresh token when another token already exist for logged in user

### ISSUE THREAD

**Issue by mdmuhtasimfuadfahim** [hagopj13/node-express-boilerplate#274]
I have hit the login API 10-15 times. All the times it creates new refresh token under tokens database. This bug should be solved as well as the #267 issue.

**Comment by chintanshahts** [comment]
It depends on your requirement. User can log in from multiple devices.
If your product does not allow the multiple device login you need to remove the refresh token else all good.

**Comment by mdmuhtasimfuadfahim** [comment]
@chintanshahts yes. But with this boilerplate there is no device information is going at the time of login. So I think this would be a good solution and please check the issue #267. To solve this issue this might help.
